src/pybrake/notifier.py

```diff
@@ -73,13 +72,19 @@ def __init__(
                 default value [re.compile("password"), re.compile("secret")]
         :param remote_config: Set up configuration from the Airbrake server,
                 default value False
+        :param max_failed_queue: Set up length of failed stats queue
+        :param manage_failed_stats: If manage_failed_stats set as true then
```

Comment:
I would call this option `backlog`. When I read:

```
* max_failed_queue
* manage_failed_stats
```

...I don't see a connection between these two options.

If I read:

```
* max_backlog_size
* backlog
```

...It's obvious there's some connection.


`backlog` can also be named `backlog_enabled`

What do you think?